tensorrt_llm/_torch/custom_ops/cute_dsl_custom_ops.py

```
@@ -2789,3 +2793,308 @@ def _(
         assert output.dtype == torch.bfloat16, "CuTe DSL fp8 bmm output dtype must be bf16"
         assert output.shape == (batch_size, m,
                                 n), "CuTe DSL fp8 bmm output shape is incorrect"
+
+    class CuteDSLFp8BlackwellGroupedGemmRunner(TunableRunner):
+        kernel_class = Sm100BlockwiseContiguousGroupedGemmKernel
+        kernel_cache = dict()
```

Ruff (0.15.0)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)